extension/js/common/composer.ts

```diff
@@ -1827,6 +1873,24 @@ export class Composer {
     }
   }
 
+  private quoteText(text: string) {
+    return text.split('\n').map(l => '<br/>&gt; ' + l).join('\n');
+  }
+
+  private formatDate(date: Date) {
```

Comment:
maybe shorten to 
```js
date.toISOString().replace(/T/, ' ').replace(/:[^:]+$/, '')
// "2019-07-02 08:15"
```
Then you could add a method as `Str.fromDate` or something like that.